src/schedule.rs

```diff
@@ -116,10 +121,10 @@ impl Schedule {
         }
 
         // We ran out of dates to try.
-        None
+        LocalResult::None
     }
 
-    fn prev_from<Z>(&self, before: &DateTime<Z>) -> Option<DateTime<Z>>
+    fn prev_from<Z>(&self, before: &DateTime<Z>) -> LocalResult<DateTime<Z>>
```

Comment:
The changes in `prev_from()` are almost the same as in `next_from()` for the same reasons.